core/CMakeLists.txt

```diff
@@ -166,6 +166,14 @@ if (libcxx AND NOT APPLE)
   target_compile_definitions(Core PRIVATE __CORRECT_ISO_CPP_WCHAR_H_PROTO)
 endif()
 
+if(MSVC)
+  # Definitions of MATH Macros like M_PI are only exposed on Windows 
```

Comment:
Could you please clarify here  `MATH` as a `MathCore` library?

Reply:
Updated. Thanks!